tensorflow/compiler/tests/BUILD

```diff
@@ -56,6 +56,7 @@ py_library(
         "//tensorflow/python/compiler/xla:compiler_py",
         "//tensorflow/python/platform:flags",
         "//tensorflow/python/platform:tf_logging",
+        "//tensorflow/python/platform:sysconfig",
```

Comment:
Why not add the dep to the `matrix_solve_op_test` build rule directly?

Reply:
There is another PR also needing this dependency (https://github.com/tensorflow/tensorflow/pull/60696). The common dependency shared by these two tests for ROCm would be xla_test. And we would anticipate that there might be more xla_test that might need to differentiate between ROCm and CUDA. So instead of adding it to the dependencies of these two tests respectively, adding it to the dependencies of xla_test would be a bit more future proof.

Reply:
Each test that uses the import line for sysconfig will need the dependency directly, not indirectly though the xla_test target. This is enforced by the strict deps check.

Reply:
Thanks for pointing it out. I have updated the PR with the change.